src/app/new-auth/components/input-field/input-field.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Input, OnInit, forwardRef } from '@angular/core';
+import { ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+/**
+ * input field for authentication module using
+ * angular material input field and designed
+ * to be used with ngModel
+ *
+ * @param pwrd: Boolean - make true if the field is for a
```

Comment:
Using isPwd is more descriptive since we are representing a boolean value

Reply:
👍